src/module/repo/ostree_repo.cpp

```diff
@@ -757,20 +757,40 @@ linglong::util::Error OSTreeRepo::pull(const package::Ref &ref, bool force)
 {
     Q_D(OSTreeRepo);
 
-    // Fixme: remote name maybe not repo and there should support multiple remote
-    return WrapError(d->ostreeRun({ "pull", d->remoteRepoName, "--mirror", ref.toString() }), "");
+    // FIXME: remote name maybe not repo and there should support multiple remote
```

Comment:
already support multiple remote repo, remove it.

Reply:
have changed